README.md

```diff
@@ -103,6 +103,32 @@ bash <(curl -L https://git.io/getLatestKialiOperator) --uninstall-mode true --op
 
 For more details on uninstalling the Kiali operator, see the [Kiali documentaton](https://www.kiali.io/documentation/getting-started/#_uninstall_kiali_operator_and_kiali).
 
+## Multitenancy
+
+The operator installs a control plane configured for multitenancy.  This installation reduces the scope of the control plane
+to only those projects/namespaces listed in a `ServiceMeshMemberRoll`.  After installing the control plane, create/update
+a ServiceMeshMemberRoll resource with the project/namespaces you wich to be part of the mesh.  The name of the
+ServiceMeshMemberRoll resource must be named `default`.  The operator will configure the control plane to watch/manage pods
+in those projects/namespaces and will configure the project/namespaces to work with the control plane.
+
```

Comment:
Does the order of creation matters? What happens if the user first creates the namespace and deploy their services and only add that namespace to a `smmr` later? Worth to document/highlight this case?

Reply:
Yes, if the application is deployed before the namespace is set up then it will not be configured for the service mesh.

Reply:
There's a typo above, wich should be wish?

Reply:
added a note about auto-injection

Reply:
So, if the user messes with the order, what should he do? Will killing the pods force the injection to happen? Must he delete the whole namespace and recreate everything again?

Reply:
yes, injection occurs on pod creation.  this should be covered by istio docs.